src/main/java/org/openpnp/machine/reference/vision/ReferenceFiducialLocator.java

```diff
@@ -177,40 +183,49 @@ public Location getHomeFiducialLocation(Location location, Part part) throws Exc
      * @return
      * @throws Exception
      */
-    private static Location getFiducialLocation(BoardLocation boardLocation, Placement fid)
-            throws Exception {
+	private static Location getFiducialLocation(BoardLocation boardLocation, Placement fid, Length fidDia) throws Exception {
         Camera camera = Configuration.get().getMachine().getDefaultHead().getDefaultCamera();
 
         Logger.debug("Locating {}", fid.getId());
 
+		Footprint footprint;
+
+		// Careful...magic number below. Also used in JobPanel. This are for
+		// global panel fiducials, which do not have an entry in the PCB BOM.
+		if ((fid.getId().equals("PanelFid1_a322") || fid.getId().equals("PanelFid2_g301"))) {
```

Comment:
This is very sensible and clear to me. I think I can do on Saturday no problem. 

So, the only change to the fidloc API would be to revert back to the old interface (before this PR) and then add a bool to indicate panel, right? And then I'd find all the places that used the existing pre-PR interface and make the change there, right? I ask because (from memory) there are drivers that are calling this interface. Just making sure I understand your direction here.

Thanks again for the well reasoned response.